### DISCUSSION TITLE
Sort by most upvoted

### DISCUSSION THREAD

**Discussion by sethfri** [community/community#12657] [Discussions]
It would be really useful to sort discussions by most upvoted. For example, take the discussions in this repo. I would like to be able to see the top voted discussions to see what GitHub is likely close to addressing, and potentially vote for those requests myself.

As things currently are, I have no idea what's popular. I have to search for keywords I might be interested and hope for the best, which means I might not discover suggestions that I wouldn't have thought of, but which I wholeheartedly agree with.

**Comment by Lee-Carre** [comment]
See #10585 & #3328.

**Comment by TPS** [comment]
@sethfri Duplicate https://github.com/orgs/community/discussions/3328 is marked <kbd>Shipped</kbd>.  Close FTW?
